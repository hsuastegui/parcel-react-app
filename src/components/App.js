import React from "react";
import Header from "./Header";
import logo from "./logo.svg";
import "./app.css"; // eslint-disable-line import/no-unassigned-import

const App = () => {
  return (
    <div className="app">
      <Header />
      <img alt="React Logo" className="appLogo" src={logo} />
      <p>Hello</p>
    </div>
  );
};

export default App;
