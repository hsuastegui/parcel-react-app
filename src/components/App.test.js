import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";
import App from "./App";

describe("<App />", () => {
  test("it works", () => {
    const wrapper = shallow(<App />);

    expect(wrapper.find(".app")).toHaveLength(1);
  });
  test("it matches snapshot", () => {
    const tree = renderer.create(<App />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
