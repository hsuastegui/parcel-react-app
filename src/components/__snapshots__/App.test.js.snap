// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> it matches snapshot 1`] = `
<div
  className="app"
>
  <header>
    Welcome
  </header>
  <img
    alt="React Logo"
    className="appLogo"
    src="logo.svg"
  />
  <p>
    Hello
  </p>
</div>
`;
